IdentityMicroService/Application/Services/Abstractions/IAuthenticationManage.cs

```diff
@@ -0,0 +1,13 @@
+﻿using IdentityMicroService.Domain.Entities.Models;
+using IdentityMicroService.Domain.Entities.Models.AuthorizationDTO;
+
+namespace IdentityMicroService.Application.Services.Abstractions
+{
+    public interface IAuthenticationService
```

Comment:
Please make the file name correspondent to the interface name (any of them is good, but if you choose the file name pay attention to change misspelles word)